lib/assessment.rb

```diff
@@ -5,7 +5,7 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum(&:size).max.last
```

Comment:
Not quite right. `enum` is an Enumerable object, not a method, so it can't accept any arguments. Instead, you'll need to call the methods that are on it.